### DISCUSSION TITLE
I'm surprised it's so fast

### DISCUSSION THREAD

**Discussion by R-Goc** [sumatrapdfreader/sumatrapdf#4355] [General]
You should add to the website that this reader is very fast, and has fast search. Compared to the free adobe acrobat (I don't know if the paid version is any better) this is way quicker. And acrobat is already faster than browser. I almost didn't download this, but now it's staying. Maybe some kind of benchmark would be good, because just saying fast and small is not enough in my opinion.

**Comment by kjk** [comment]
Well, the second line on [website](https://www.sumatrapdfreader.org/free-pdf-reader) is: `Fast, small, packed with features, customizable, free.`

But ultimately I think people just need to try it for themselves to feel the difference.

**Comment by R-Goc** [comment]
Have you considered or benchmarked optimizing for speed instead of size? Using /O2 instead of /O1?

**Reply by kjk** [reply]
I haven't and I don't have the bandwidth for it right now. If you feel like doing that and posting the results...
